Makefile

```
@@ -47,7 +47,7 @@ build_dev: parser-messages
 	dune build $(COMPILER_DIR)/catala.exe $(BUILD_SYSTEM_DIR)/clerk.exe
 
 #> build					: Builds the Catala compiler
-build: parser-messages format
+build: parser-messages format plugins
 	dune build $(COMPILER_DIR)/catala.exe $(BUILD_SYSTEM_DIR)/clerk.exe
```

Better handle the dependency through dune (so that we can run it just once):
```suggestion
build: parser-messages format 
	dune build $(COMPILER_DIR)/catala.exe $(COMPILER_DIR)/plugins/ $(BUILD_SYSTEM_DIR)/clerk.exe
```